lib/experimental/fonts/font-library/font-library.php

```diff
@@ -45,26 +45,25 @@ function gutenberg_init_font_library_routes() {
 			'delete_published_posts' => 'edit_theme_options',
 		),
 		'map_meta_cap'                   => false,
+		'query_var'                      => false,
+		'show_in_rest'                   => true,
 		'rest_base'                      => 'font-families',
 		'rest_controller_class'          => 'WP_REST_Font_Families_Controller',
-		'autosave_rest_controller_class' => 'WP_REST_Autosave_Font_Families_Controller',
-		'query_var'                      => false,
+		'autosave_rest_controller_class' => 'WP_REST_Autosave_Fonts_Controller',
 	);
```

Comment:
Ordering the post type settings for `wp_font_family` o they match with `wp_font_face`.